app/models/patient.rb

```diff
@@ -173,7 +173,9 @@ class Patient < ApplicationRecord
   has_many :transfers
   has_many :contact_attempts
   has_many :common_exposure_cohorts
+  # rubocop:enable Rails/HasManyOrHasOneDependent
```

Comment:
The options for `:dependent` are listed [here](https://guides.rubyonrails.org/association_basics.html#options-for-has-one-dependent). There is no option we can use that will maintain the same behavior we have here (which is to do nothing). Since the patient is such an essential piece, I was wary to make such a major change to how we handle dependent objects. I think if we want to make this change, we should do it in a separate PR so that we can do our due diligence determining:
1. Which dependent type we really want
2. If changing the dependent type is going to cause any performance differences when deleting patients

Does that make sense? If so I can make a new ticket in the backlog. If not and you're not as worried, I can add a dependent type here.

Reply:
You can also explicitly specify
>[`nil` do nothing (default).](https://api.rubyonrails.org/classes/ActiveRecord/Associations/ClassMethods.html#method-i-has_many-label-Options)

which is what Kyle had done on some other models, like Jurisdiction. It's also low risk because we never destroy Patients.

That said, you're right that we should probably consider a strategy better than doing nothing, but that can be a future ticket.

Reply:
Oops didn't realize you could do `nil`. Updated to do that in https://github.com/SaraAlert/SaraAlert/pull/1147/commits/043a7b7e122887ad4e5a760533265982831e3ce1.

Reply:
https://tracker.codev.mitre.org/browse/SARAALERT-1644